sdk/python/tests/run_test_server.py

```diff
@@ -452,12 +453,16 @@ def run_nginx():
 
     env = os.environ.copy()
     env['PATH'] = env['PATH']+':/sbin:/usr/sbin:/usr/local/sbin'
+    os.mkfifo(nginxconf['ACCESSLOG'], 0700)
```

Comment:
Good catch! I didn't notice because I'm testing in docker containers that have been built from scratch each time. I think the right way to fix this is probably to clean up by removing the ACCESSLOG fifo after the test. I thought about just checking if the fifo exists and only making it if necessary, but that could lead to trouble if a test is interrupted in such a way that the cat doesn't get killed (and in that case, it should be an error, but it can print a more informative message).